django/pyrog/models.py

```diff
@@ -0,0 +1,180 @@
+from django.db import models
+
+from cuid import cuid
+
+# NOTE:
+# * Some FK fields are expected to be sometimes null in the previous DB schema.
+# * User, Comment and AccessControl tables will be added. The User table should be
+#   implemented with respect to the framework (django).
+# * Everything is now snake-cased. Other cases could be provided through serialization.
+
+
+class Template(models.Model):
+    id_ = models.TextField(name="id", primary_key=True, default=cuid, editable=False)
+    name = models.TextField(unique=True)
+
+    updated_at = models.DateTimeField(auto_now=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+
+class Source(models.Model):
+    class Meta:
+        unique_together = (("name", "template"),)
+
+    id_ = models.TextField(name="id", primary_key=True, default=cuid, editable=False)
+    template = models.ForeignKey(Template, related_name="sources", on_delete=models.CASCADE)
+    name = models.TextField()
+    version = models.TextField(blank=True, default="")
+
+    updated_at = models.DateTimeField(auto_now=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+
+class Resource(models.Model):
+    id_ = models.TextField(name="id", primary_key=True, default=cuid, editable=False)
+    source = models.ForeignKey(Source, related_name="resources", on_delete=models.CASCADE)
+    label = models.TextField(blank=True, default="")
+    primary_key_table = models.TextField()  # Expected to sometimes be null in prod
+    primary_key_column = models.TextField()  # Expected to sometimes be null in prod
```

Comment:
may be null in prod but should not in the new model (getting rid of resources with empty values during migration is ok)